snap/snapcraft.yaml

```diff
@@ -90,6 +90,18 @@ parts:
     stage:
     - -linux
     - -src
+    override-build: |
+      snapcraftctl build
+      SO_FILE_32=$(find ../install/linux/lib32/* -name "libphobos2.so.[0-9]*.[0-9]*.[0-9]*")
```

Comment:
I'm afraid this fails for me when I run in `snapcraft cleanbuild`:

```
mkdir -p ../install/src/phobos/etc
mkdir -p ../install/src/phobos/std
cp -r std/* ../install/src/phobos/std/
cp -r etc/* ../install/src/phobos/etc/
cp LICENSE_1_0.txt ../install/phobos-LICENSE.txt
make[1]: Leaving directory '/root/build_dmd/parts/phobos/build'
find: ‘../install/linux/lib32/*’: No such file or directory
Failed to run 'override-build': Exit code was 1.
```

Reply:
oh, ok now it works